discovery-provider/scripts/start.sh

```diff
@@ -106,20 +112,16 @@ if [[ "$audius_discprov_dev_mode" == "true" ]]; then
         echo "Finished running migrations"
     fi
 
-    # filter tail to server/worker/beat logs with
-    # docker exec -it <container> tail -f /var/log/discprov-server.log
-    # docker exec -it <container> tail -f /var/log/discprov-worker.log
-    # docker exec -it <container> tail -f /var/log/discprov-beat.log
-    ./scripts/dev-server.sh 2>&1 | tee >(logger -t server) &
+    audius_service=server ./scripts/dev-server.sh 2>&1 | tee >(logger -t server) &
```

Comment:
Only caveat is that by following this `2>&1` pattern, all stderr output is converted to stdout output.

This is the reason why our `syslog.severity` tags within Loggly show up as `informational` even though they are errors.

More work can be done to have errors stay on stderr, but it is not the worth the time commitment since `json.logLevel` will show us `info` vs `error` even if `syslog.severity` is always `informational`.

Reply:
hadn't thought about this but this makes sense. is there a way we could pipe both separately to loggly using rsyslog?

Reply:
I can take an hour and see how best to get this done in a way that's not too hacky for the discovery provider, but I would vote leaving the rsyslog implementation as is since Logspout is currently the new way forward.

Personally, I think the syslog key should be ignored completely since we should be looking at the json key for information from the container.

Additionally, because we use bunyan, I thought I saw all logs always hitting stdout. If correct, we might have to again do something hacky with bunyan to get bunyan to output to stderr as well.

Thoughts?

Reply:
yeah not a blocker for now, lets leave as is